docs/builtins/General/enforce.md

```diff
@@ -18,7 +18,7 @@ Use the following arguments to specify the test expression and error message for
 
 | Argument | Type   | Description                                    |
 |----------|--------|------------------------------------------------|
-| `expression` | bool | Specifies the expression to evaluate.     |
+| expression | bool | Specifies the expression to evaluate.     |
```

Comment:
@lsgunnlsgunn last PR's added the backticks. What is the _correct_ style now, see: https://github.com/kadena-io/pact-5/pull/278/files

Reply:
I don't actually understands why this breaks docusaurus, but it looks weird in our ansi terminal renderer, but lisa can add them back if necessary

Reply:
We don't need backticks around ordinary words or words to indicate syntax, per se. We only *require* them for certain characters that freak Docusaurus out where the docs engine doesn't know how to interpret things like < and > or { and } except as HTML. I don't know what all the characters are that it doesn't like. Amir put something together that records them, I think.